src/plugins/ceramic-feed/server/content-types/ceramic-post/lifecycles.js

```diff
@@ -154,7 +156,26 @@ const LifecycleHooks = new class {
     const { relatedFieldName } = metadata
     const entitySchema = schemaName ? schemas[schemaName] : schema
     const { fields, mediaFields, relationFields } = entitySchema
-    const payload = pick(entity, fields)
+
+    const payload = pick(
+      entity,
+      array.remove(fields, 'publishWallet', 'publishDapp') //converting to tags
```

Comment:
let's define those tags (& titles) as some const array

then we could just refer to it which will simplify adding new tags (if need)

Reply:
done